Sources/Build/BuildOperation.swift

```diff
@@ -950,19 +1040,111 @@ extension BuildOperation {
     }
 }
 
+extension BuildOperation {
+    fileprivate typealias PluginTool = (path: AbsolutePath, triples: [String]?)
```

Comment:
I'd prefer this to be a separate struct with detailed doc comments on both properties.

Reply:
Oh, sure, I can do that!